### PR TITLE
upgrade to new version

### PR DESCRIPTION
**Pull request by dmpe**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN apt-get -qq update \
 	&& rm -rf /var/lib/apt/lists/*
 
 # Download and install hugo
-ENV HUGO_VERSION 0.53
+ENV HUGO_VERSION 0.54
```
Comment (omniwired) [inline Dockerfile]: in my experience, it needs to be 0.54.0 for this one

Comment (dmpe) [reply Dockerfile]: thank you @omniwired

**Comment by dmpe** [comment]
@oskapt well, thanks!

**Comment by oskapt** [comment]
Sorry, @dmpe, I didn't see your PR. Github is flaky about sending me notifications. I saw that you closed this, but I never received a notification that you submitted it. I'll look into that.

I follow the exact release numbers from hugo's github page. This is the first time they've added `.0` after the release, but since they did it, I have to do it too. The `Dockerfile` uses the release number to create the source location from which to pull the `.deb`, and that path has `0.54.0` in it.

Thanks for submitting this, and again, sorry I didn't see it.

I'm building `0.54.0-1` now which changes the source location from which I pull the file. I discovered, thanks to this PR, that it was still using upstream's location instead of `gohugoio` directly. That's fixed now, and you have my thanks for helping me discover it.

**Comment by dmpe** [comment]
@oskapt Oh, no problem at all 🥂  - I am glad that you can now see people's PR and that everything is solved. 👍

**Pull request closed without merging**